### PR TITLE
Homework task number 2 - done

### LINKED ISSUE #1: #1 Cargo, operator==
Zadanie 1

W klasie Cargo napisz operator porównania (operator==), który będzie sprawdzał, czy towary są takie same.

### PR DESCRIPTION
**Pull request by MarekJanek**
Actually it's task 1.

### CODE REVIEW

File: shm/cargo.hpp [hunk 1]
```diff
@@ -16,6 +16,12 @@ class Cargo {
         return *this;
     }
 
+    bool operator==(const Cargo& freight2) {
+        if (name_ == freight2.name_)
```
Comment (majkel84) [inline shm/cargo.hpp]:
```suggestion
        return name_ == freight.name_;
```

**Review by majkel84 — COMMENTED** [review]
sugest return statement change as in example
https://stackoverflow.com/questions/7046120/returning-value-in-conditional-operator

File: shm/cargo.hpp (hunk 1 above)
Comment (MarekJanek) [reply shm/cargo.hpp]: Great! Could you please change freight to freight2.name_ ? Actually freight2 is variable name, then I'll accept that ! :)

Comment (majkel84) [inline shm/cargo.hpp]:
```suggestion
        return name_ == freight2.name_;
```

**Review by majkel84 — CHANGES_REQUESTED** [review]

File: shm/cargo.hpp [hunk 2]
```diff
@@ -16,6 +16,12 @@ class Cargo {
         return *this;
     }
 
+    bool operator==(const Cargo& freight2) {
+        return name_ == freight2.name_;
+            return true;
```
Comment (majkel84) [inline shm/cargo.hpp]:
```suggestion

```

File: shm/cargo.hpp [hunk 3]
```diff
@@ -16,6 +16,12 @@ class Cargo {
...
+        return false;
```
Comment (majkel84) [inline shm/cargo.hpp]:
```suggestion

```

**Review by majkel84 — CHANGES_REQUESTED** [review]

**Review by majkel84 — APPROVED** [review]

**Pull request merged**